torch/_inductor/config.py

```diff
@@ -167,7 +167,7 @@ def fx_graph_remote_cache_default():
 
 # Post grad fusion and options, set to empty dict to disable fusion.
 # Call `torch._inductor.fx_passes.group_batch_fusion.list_group_batch_fusions(False)` to see available fusions.
-post_grad_fusion_options: Dict[str, Dict[str, Any]] = {}
+post_grad_fusion_options: Dict[str, Dict[str, Any]] = {"b2b_gemm_pass": {}}
```

Comment:
can you put this as a separate config option (defaulted to false)? if someone passes in `post_grad_fusion_options` this will get overriden

Reply:
added a separate config option